common/src/JetCorrections.cxx

```diff
@@ -1072,12 +1072,15 @@ std::unique_ptr<FactorizedJetCorrector> build_corrector(const std::vector<std::s
   //propagate to MET
   //apply type1 MET correction to RAW MET
   //NB: jet with substracted muon Pt should be used
-  void correct_MET(const Event & event, const bool & do_L1corr){
+  void correct_MET(const Event & event, const bool & do_L1corr, double pt_thresh, double eta_thresh_low, double eta_thresh_high){
     //we start from raw MET
     LorentzVector metv4= event.met->uncorr_v4();
     for(auto & jet : *event.jets){
       //thresholds on the corrected jets: pt > 15, EM fraction < 0.9
-      if(jet.v4().Pt() > 15 && (jet.neutralEmEnergyFraction()+jet.chargedEmEnergyFraction())<0.9){
+      bool to_be_corrected = jet.v4().Pt() > 15.;
+      to_be_corrected = to_be_corrected && ( fabs(jet.v4().Eta())<eta_thresh_low || fabs(jet.v4().Eta())>eta_thresh_high || jet.v4().Pt() > pt_thresh );
```

Comment:
Are these meant to be all ORs? e.g. if you had `eta_thres_low = 1`, `eta_thres_high=3`, `pt_thres=20`, a jet with eta = 0.1 and pt = 18 would still pass selection. Would have thought you want to pass the eta AND pt requirements?

Reply:
Also I assume the intention is to select jets _outside_ or a certain eta window, and not inside?

Reply:
the intention is NOT to correct jets in a eta-window below a certain pt-threshhold